lib/src/base-styles/_nx-radio-checkbox.scss

```diff
@@ -104,11 +104,11 @@ svg.nx-radio-checkbox__control {
 
 .nx-radio-checkbox__input {
   margin: 0;
-  width: 0;
 
   // in FF, the checkboxes don't respect the width: 0 and we need this in order to hide them
   // Fix discovered via the title of https://bugzilla.mozilla.org/show_bug.cgi?id=605985
```

Comment:
I think at this point this comment can just be removed.  We are consistently using `appearance` to hide the native input so this comment about firefox specifics isn't important.

Reply:
> I think at this point this comment can just be removed. We are consistently using `appearance` to hide the native input so this comment about firefox specifics isn't important.

Removed.
f7ab44bcecb80e17b5fa8b7859baf0de80fad44f